documentation/book/con-kafka-listeners.adoc

```diff
@@ -37,56 +37,13 @@ listeners:
 
 == External listener
 
-The external listener is used to connect to a Kafka cluster from outside of an {ProductPlatformName} environment.
-{ProductName} supports three types of external listeners:
+The external listener is used to connect to a Kafka cluster from outside of a Kubernetes environment.
+{ProductName} supports the following types of external listener:
```

Comment:
I think it should be singular because we only support configuring a single external listener (even though there are multiple types to chose from). @PaulRMellor do you agree, or should we reword this?